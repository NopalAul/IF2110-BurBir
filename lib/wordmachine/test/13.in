13
13213123;
