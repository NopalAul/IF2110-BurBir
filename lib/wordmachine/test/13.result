lib/wordmachine/test/13.in lib/wordmachine/test/13.out: WRONG
