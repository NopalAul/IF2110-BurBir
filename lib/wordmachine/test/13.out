13213123
1
TRUE
